docker-compose.yaml

```diff
@@ -0,0 +1,419 @@
+  #
+  # 'GF_AUTH_PROXY_ENABLED=true'
+  # 'GF_AUTH_PROXY_HEADER_NAME='X-Forwarded-User'
+  # 'GF_SERVER_ROOT_URL='https://grafana.example.com'
+  grafana:
+    container_name: grafana
+    image: 'index.docker.io/sourcegraph/grafana:10.0.7@sha256:c171332bccb1e01a54411f669b31e7c46d2a5568eac075bf05e4be0229ec262a'
+    volumes:
+      - '~/sourcegraph-docker/grafana-disk:/var/lib/grafana'
+      - './grafana/datasources:/sg_config_grafana/provisioning/datasources'
+      - './grafana/dashboards:/sg_grafana_additional_dashboards'
+    ports:
+      - '0.0.0.0:3370:3370'
+    networks:
+      - sourcegraph
+    restart: always
+
+  # Description: Publishes Prometheus metrics about Docker containers.
+  #
+  # Disk: none
+  # Network: 100mbps
+  # Liveness probe: none
+  # Ports exposed to other Sourcegraph services: 8080/TCP
+  # Ports exposed to the public internet: none
+  #
+  cadvisor:
+    container_name: cadvisor
+    image: 'google/cadvisor:v0.33.0'
+    volumes:
+      - '/:/rootfs:ro'
+      - '/var/run:/var/run:ro'
+      - '/sys:/sys:ro'
+      - '/var/lib/docker/:/var/lib/docker:ro'
+      - '/dev/disk/:/dev/disk:ro'
+    networks:
+      - sourcegraph
+    restart: always
+
+  # Description: PostgreSQL database for various data.
+  #
+  # Disk: 128GB / persistent SSD
+  # Network: 100mbps
+  # Liveness probe: 5432/TCP
+  # Ports exposed to other Sourcegraph services: 5432/TCP 9187/TCP
+  # Ports exposed to the public internet: none
+  #
+  pgsql:
+    container_name: pgsql
+    image: 'index.docker.io/sourcegraph/postgres-11.4:19-11-14_b084311b@sha256:072481559d559cfd9a53ad77c3688b5cf583117457fd452ae238a20405923297'
+    volumes:
+      - '~/sourcegraph-docker/pgsql-disk:/data/pgdata'
+    networks:
+      - sourcegraph
+    restart: always
+
+  # Description: Redis for storing short-lived caches.
+  #
+  # Disk: 128GB / persistent SSD
+  # Network: 100mbps
+  # Liveness probe: 6379/TCP
+  # Ports exposed to other Sourcegraph services: 6379/TCP 9121/TCP
+  # Ports exposed to the public internet: none
+  #
+  redis-cache:
+    container_name: redis-cache
+    image: index.docker.io/sourcegraph/redis-cache:19-04-16_6891de82@sha256:4cbfac8af0abb673899250d4fd859cc477d6426de519e9deb71e454e18322499
+    volumes:
+      - '~/sourcegraph-docker/redis-cache-disk:/redis-data'
+    networks:
+      - sourcegraph
+    restart: always
+  # Description: Redis for storing semi-persistent data like user sessions.
+  #
+  # Disk: 128GB / persistent SSD
+  # Network: 100mbps
+  # Liveness probe: 6379/TCP
+  # Ports exposed to other Sourcegraph services: 6379/TCP 9121/TCP
+  # Ports exposed to the public internet: none
+  #
+  redis-store:
+    container_name: redis-store
+    image: 'index.docker.io/sourcegraph/redis-store:19-04-16_6891de821@sha256:56426d601ce1f6d63088fea1cefa61f69a2e809c7d90fc1d157cca63cf81b277'
+    volumes:
+      - '~/sourcegraph-docker/redis-store-disk:/redis-data'
+    networks:
+      - sourcegraph
+    restart: always
```

Comment:
This file is missing jaeger services. They should be added and commented out, so that we at least keep things in sync with one-another.